File: PHP/UberBlack.php
<?php
require_once('Car.php');
//Clase UberBlack (Hereda de la clase Carro)
class UberBlack extends Car
{
    // Atributos o propiedades de la clase
    public $typeCarAccepted;
    public $seatsMaterial;

    // Método constructor
    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial)
    {
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

//Clase UberBlack (Hereda de la clase Carro)
class UberBlack extends Car {
    // Atributos o propiedades de la clase
    private Integer id;
    private Map<String, ArrayList<String, Integer>> typeCarAccepted;
    private ArrayList<String> seatsMaterial;

    // Método constructor
    public UberBlack(String license, Driver driver, Map<String, ArrayList<String, Integer>> typeCarAccepted,
            ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}